src/components/Calculator.js

```diff
@@ -1,71 +1,57 @@
-        <button type="button" className="btn-add orange" onClick={this.handleClick} onKeyDown={this.handleClick}>+</button>
-        <button type="button" className="btn-subtract orange" onClick={this.handleClick} onKeyDown={this.handleClick}>-</button>
-        <button type="button" className="btn-multiply orange" onClick={this.handleClick} onKeyDown={this.handleClick}>x</button>
-        <button type="button" className="btn-divide orange" onClick={this.handleClick} onKeyDown={this.handleClick}>÷</button>
-        <button type="button" className="btn-percent light-gray" onClick={this.handleClick} onKeyDown={this.handleClick}>%</button>
-        <button type="button" className="btn-1 light-gray" onClick={this.handleClick} onKeyDown={this.handleClick}>1</button>
-        <button type="button" className="btn-2 light-gray" onClick={this.handleClick} onKeyDown={this.handleClick}>2</button>
-        <button type="button" className="btn-3 light-gray" onClick={this.handleClick} onKeyDown={this.handleClick}>3</button>
-        <button type="button" className="btn-4 light-gray" onClick={this.handleClick} onKeyDown={this.handleClick}>4</button>
-        <button type="button" className="btn-5 light-gray" onClick={this.handleClick} onKeyDown={this.handleClick}>5</button>
-        <button type="button" className="btn-6 light-gray" onClick={this.handleClick} onKeyDown={this.handleClick}>6</button>
-        <button type="button" className="btn-7 light-gray" onClick={this.handleClick} onKeyDown={this.handleClick}>7</button>
-        <button type="button" className="btn-8 light-gray" onClick={this.handleClick} onKeyDown={this.handleClick}>8</button>
-        <button type="button" className="btn-9 light-gray" onClick={this.handleClick} onKeyDown={this.handleClick}>9</button>
+    if (nullCheck || !Object.keys(objData).length) {
+      updateScreen('0');
+    } else if (!objData.next) {
+      updateScreen(objData.total);
+    } else updateScreen(objData.next);
+
+    updateData({ ...data, ...objData });
+  };
+
+  return (
+    <div className="calculator-container">
+      <div className="screen">
+        <div>{screen}</div>
       </div>
-    );
-  }
-}
+      <button type="button" className="btn-reset light-gray" onClick={handleClick} onKeyDown={handleClick}>AC</button>
+      <button type="button" className="btn-change light-gray" onClick={handleClick} onKeyDown={handleClick}>+/-</button>
+      <button type="button" className="btn-0 light-gray" onClick={handleClick} onKeyDown={handleClick}>0</button>
+      <button type="button" className="btn-dot light-gray" onClick={handleClick} onKeyDown={handleClick}>.</button>
+      <button type="button" className="btn-equal orange" onClick={handleClick} onKeyDown={handleClick}>=</button>
+      <button type="button" className="btn-add orange" onClick={handleClick} onKeyDown={handleClick}>+</button>
```

Comment:
- [ ] OPTIONAL: This could have been a separate component (`Buttons`) in a separate file and rendered here. Smaller components are easy to fix, reusable, and make the app more scalable. :pray:

Reply:
@moise10r Thank you for taking the time to leave this comment and for your detailed feedback. I really appreciate it. I am planning to refactor this part as you advised after I finish my work for this week.
Thank you once again!